tests/mgbench/benchmark.py

```diff
@@ -161,7 +119,19 @@ def parse_args():
     benchmark_parser.add_argument(
         "--export-results",
         default=None,
-        help="file path into which results should be exported",
+        help="file path into which results for in_memory_transactional storage mode should be exported",
+    )
+
+    benchmark_parser.add_argument(
+        "--export-results-in-memory-analytical",
+        default=None,
+        help="file path into which results for in_memory_analytical storage mode should be exported",
```

Comment:
I can see that the definition of --export-results... defines if something is being run. Can you please update the help messages to describe this?